osi_common.proto

```diff
@@ -215,6 +215,39 @@ message Identifier
     optional uint64 value = 1;
 }
 
+// \brief References to external objects
+//
+// The external reference is used to identify objects defined in non-OSI
+// descriptions. This could be other OpenX-Standards as well as user defined
+// simulation environments.
+//
+message ExternalReference
+{
+    // The source of the external references
+    //
+    // Defines the original source of an object as an URI.
+    // The URI should be follow the syntax according to 
+    // \link https://tools.ietf.org/html/rfc3986 RFC 3986\endlink.
+    //
+    optional string reference = 1;
+    
+    // The type of the external references
+    //
+    // Can be used to describe the original source, e.g. OpenDRIVE 1.6
+    //
+    optional string type = 2;    
+    
+    // The external identifier reference value.
+    //
+    // For a common description of the external identifier, where a wide range
+    // of identification types could be represented, the repeded string is chosen.
+    //
+    // E.g. referencing a unique lane in OpenDRIVE 
+    // (RoadId --> String, S-Value of LaneSection --> Double, LaneId --> Int)
+    // 
+    repeated string identifier = 3;
```

Comment:
Add a note maybe:
More details on how identifiers are set to be used for referencing external objects are given in places where the ExternalIdentifier message is used.